### DISCUSSION TITLE
custom unit model question

### DISCUSSION THREAD

**Discussion by finAtoZ** [IDAES/idaes-pse#1261] [Q&A]
Hello all，

I‘m trying to create a new unit model. I find that there is a line that “m.fs.compressor.inlet.flow_mol.fix(1)”. If I want to add new parameter like “m.fs.compressor.inlet.***”, what should I do? A single example would be greatly helpful. Thanks!

**Comment by andrewlee94** [comment]
@finAtoZ It is not clear from your question exactly what you are trying to do here.

If you are asking about adding additional members to the port (`inlet`) then the answer is that you cannot do that in the unit model - Port members are defined only by the property package and due to the need to ensure the problem is well-posed you cannot add arbitrary members to a Port.
